core/sql/optimizer/NATable.cpp

```diff
@@ -5110,6 +5138,12 @@ NATable::NATable(BindWA *bindWA,
     setIsExternalTable(TRUE);
   }
  
+  if (qualifiedName_.getQualifiedNameObj().isHistograms() || 
+      qualifiedName_.getQualifiedNameObj().isHistogramIntervals())
```

Comment:
CmpSeabaseDDL.h has a static method called isHistogramTable that checks for both histograms and histogram_intervals.  I might be better to call this in case you ever add another histograms table.